tests/expense.test.js

```diff
@@ -2,12 +2,16 @@
 
 const supertest = require('supertest');
 const { createServer } = require('../src/createServer');
+const { resetMockUsers } = require('../src/routes/usersRouter');
+const { resetMockExpenses } = require('../src/routes/expensesRouter');
 
 describe('Expense', () => {
   let server;
   let api;
 
   beforeEach(() => {
+    resetMockExpenses();
+    resetMockUsers();
```

Comment:
You could actually prevent this if your Repository were a Class.
But that is another topic xD, you don't have to change it here